main.tf

```diff
@@ -56,6 +56,11 @@ module "database" {
   depends_on = [module.networking]
 }
 
+moved {
+  from = module.redis.azurerm_redis_cache.default
+  to   = module.redis[0].azurerm_redis_cache.default
+}
```

Comment:
IIRC from previous rollouts moving an item into a count state is a bit a bit of a break change, please make sure the major version get bumped.